ci-operator/step-registry/rosa/cluster/wait-ready/nodes/rosa-cluster-wait-ready-nodes-commands.sh

```diff
@@ -150,6 +150,19 @@ function getDesiredComputeCount {
     echo "--auto-scaling disabled, --replicas set, retrieving replica count desired"
     desired_compute_count=$(rosa describe cluster -c "$CLUSTER_ID"  -o json  |jq -r '.nodes.compute')
   fi
+  if [[ ! -z ${MP_NAME} ]];then
```

Comment:
The variable is not defined. Need to update the machinepool step, and get it from there.